pkg/util/consts/consts.go

```diff
@@ -0,0 +1,54 @@
+package consts
```

Comment:
this should be in a test package or something otherwise it will just pollute everything